### DISCUSSION TITLE
【keyball44】キーソケットのランドが剥離してしまいました。

### DISCUSSION THREAD

**Discussion by Laplus8** [Yowkees/keyball#295] [Q&A]
お世話になっております。
27番のキースイッチは無事動くようになりましたが、16番の反応が悪く左側のランドも剥離しているかもしれません。
再度解決策をご教授頂けますと幸いです。
重ねてで申し訳ありませんがお時間のある時にご確認の程、お願い致します。

![image](https://github.com/Yowkees/keyball/assets/139874476/823dd1be-3355-48a4-a23e-0af7cc3fe2df)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
16番の右側の配線は合っています。左側の端子は下記画像の緑点どうしを接続していただければ復活すると思います。
お手数ですがご対応お願いいたします。
![KB44_Keyfix2](https://github.com/Yowkees/keyball/assets/84209772/3cd00f73-c3a2-4348-8bb3-3281b11234e9)

**Reply by Laplus8** [reply]
本日ご解説頂いた場所を繋ぎ無事に使えるようになりました！
ありがとうございました！

**Comment by Yowkees** [comment]
無事に直ったようで良かったです！
ぜひ末永くご愛用いただければ幸いです。ありがとうございました！
